javascript/libs/ui/src/lib/inputs/checkbox/checkbox.component.html

```diff
@@ -0,0 +1,22 @@
+<!--
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+Copyright (c) 2023-present Kaleidos INC
+-->
+
+<div class="checkbox-container">
+  <input
```

Comment:
Is it necessary to put cursor pointer when user do hover on the checkbox?

Reply:
Probably not because the default checkbox doesn't have one. But I think is a good idea.